handlers/agentinfo/info.go

```diff
@@ -122,6 +125,8 @@ func (ai *agentInfo) RecordOpData(latency time.Duration, payloadBytes int, err e
 		ai.nextUpdate = now.Add(updateInterval)
 	}
 
+	stats.ImdsFallbackSucceed = ai.imdsFallbackSucceed()
```

Comment:
The metadata provider `InstanceID` and `Hostname` functions are not called anywhere outside of unit tests. The provider `Get` function is only called once upon the processor `Start` function, which is called before any `RecordOpData` call. Could we use a `sync.Once` here to only set this field only one time?

Reply:
Do you mean as apart from the enabled once code you have when creating a new agent info? That happens in the start func of cw. I think that will have a  thread timing issue.

Reply:
Talked with @bryce-carey on this over slack. Changed to only publish this metric 1 time per minute. No need for single sync.